classes/export/csv/cells/cell_base.php

```diff
@@ -250,7 +250,7 @@ public function get_stage_identifier_for_assessor($submission, $student) {
      * Function to validate cell for the file upload
      * @return mixed
      */
-    public function validate_cell($value, $submissions, $stage_dentifier='', $uploadedgradecells  = []) {
+    public function validate_cell($value, $submissions, $stagedentifier='', $uploadedgradecells  = []) {
```

Comment:
dentifier again

Reply:
These look like typos in the original code.  I need to do another PR to clear up the files I left to avoid conflict to will fix those in there